Tests/SwerverTests/RouterSpec/RouterSpec.swift

```diff
@@ -6,122 +6,170 @@ import Foundation
 class RouterSpec: QuickSpec {
     override func spec() {
         describe("A Router") {
-            var router = Router()
-
-            guard let urlOne = URL(string: Resource.test_get.rawValue),
-                let urlTwo = URL(string: Resource.test_head.rawValue),
-                let urlThree = URL(string: Resource.test_option.rawValue),
-                let urlNotFound = URL(string: "/not_here") else {
-                return
+            func buildRouter() -> Router {
+                return Router()
             }
 
-            context("when receive an invalid request object") {
-                let anInvalidRequest: HttpRequest? = nil
-                let expectedResponse = router.process(request: anInvalidRequest)
-                
+            context("receives an invalid request object") {
                 it("returns a response with a 404 status code") {
```

Comment:
Try updating these tests to follow the Arrange/Act/Assert pattern. See here: http://wiki.c2.com/?ArrangeActAssert

Reply:
will take a look

Reply:
implemented